web/templates/settings/attachments_settings.hbs

```diff
@@ -2,7 +2,7 @@
     <div id="attachment-stats-holder"></div>
     <div class="settings_panel_list_header">
         <h3>{{t "Your uploads"}}</h3>
-        {{> filter_text_input id="upload_file_search" placeholder=(t 'Filter uploaded files') aria_label=(t 'Filter uploads')}}
+        {{> filter_text_input id="upload_file_search" placeholder=(t 'Filter') aria_label=(t 'Filter uploads')}}
```

Comment:
I think we can get right of the `aria_label` on these, given the heading just above it explaining what this filter input is for? Doing so will save work for translators.